### ISSUE TITLE
Error on /admin/config/people/emails

### ISSUE THREAD

**Issue by bd0bd** [backdrop-contrib/privatemsg#6]
If you press SAVE CONFIGURATION button (even without any changes done) at /admin/config/people/emails then we get a lot of errors.
This happens because of Privatemsg and not because of other modules (I have tested).

```
The Subject is using the following invalid tokens: [site:name].
The Body is using the following invalid tokens: [user:name], [user:one-time-login-url], [site:name], [site:login-url].
The Subject is using the following invalid tokens: [user:name], [site:name].
The Body is using the following invalid tokens: [user:name], [site:name].
The Subject is using the following invalid tokens: [user:name], [site:name].
The Body is using the following invalid tokens: [user:name], [user:one-time-login-url], [site:name], [site:login-url].
The Subject is using the following invalid tokens: [user:name], [site:name].
The Body is using the following invalid tokens: [user:name], [user:one-time-login-url], [site:name].
The Subject is using the following invalid tokens: [user:name], [site:name].
The Body is using the following invalid tokens: [user:name], [user:one-time-login-url], [site:name], [site:login-url].
The Subject is using the following invalid tokens: [user:name], [site:name].
The Body is using the following invalid tokens: [user:name], [site:name].
The Subject is using the following invalid tokens: [user:name], [site:name].
The Body is using the following invalid tokens: [user:name], [user:cancel-url], [site:name], [site:url:brief].
The Subject is using the following invalid tokens: [user:name], [site:name].
The Body is using the following invalid tokens: [user:name], [site:name].
```

**Comment by Graham-72** [comment]
This is generated by core's user module and not privatemsg.
We need to set the tokens [user:name] [site:name] etc, but how?

**Comment by bd0bd** [comment]
When I have uninstalled Privatemsg then everything was ok.

**Comment by Graham-72** [comment]
OK.
I note that when Privatemsg is installed we no longer get shown tokens when clicking 'browse available tokens' on the account emails page. I will try and find out what has happened.

**Comment by Radcliffe** [comment]
Fixed by PR #12.
